rise-data.html

```diff
@@ -120,7 +120,7 @@
       _getCacheUrl: function( key, method ) {
         var url = this._baseCacheUrl + "/" + this.endpoint;
 
-        if ( method === "PUT" || method === "DELETE" || method === "GET" ) {
+        if ( method === "DELETE" || method === "GET" ) {
```

Comment:
@tejohnso Removing "PUT" from acceptable methods for appending the `"/" + key` to the url below.